wordle/src/main/kotlin/domain/Answer.kt

```diff
@@ -0,0 +1,20 @@
+package domain
+
+import java.time.LocalDate
+import java.time.Period
+
+class Answer(val value: String) {
```

Comment:
```suggestion
class Answer(
    val value: String
) {
```

보통은 프로퍼티를 이렇게 쓰더라고요?

Reply:
추가로 Answer가 지금은 단어 풀에서 뽑아오지만, 직접 입력하거나 할 수도 있을 것 같은데, 그럴 때를 대비해서 Word의 규칙을 공유하도록 Word를 프로퍼티로 가지는 것은 어떨까요?